uaclient/conftest.py

```diff
@@ -108,4 +108,7 @@ def for_attached_machine(
             config.write_cache("machine-token", machine_token)
             return config
 
+        def override_features(self, features_override):
+            self.cfg.update(features_override)
```

Comment:
Since we know we are overriding features, let's make this  the following to reduce the duplication when calling this function and forcing all callsites to provide a dict with a top-level  {"features": } key
```suggestion
            self.cfg.update({"features": features_override})
```